tests/e2e/specs/frontend/checkout.test.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * External dependencies
+ */
+import {
+	merchant,
+	createSimpleProduct,
+	setCheckbox,
+	settingsPageSaveChanges,
+	verifyCheckboxIsSet,
+} from '@woocommerce/e2e-utils';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getNormalPagePermalink,
+	shopper,
+	visitPostOfType,
+} from '../../../utils';
+
+const block = {
+	name: 'Checkout',
+};
+
+const simpleProductName = 'Woo Single #1';
+const singleProductPrice = '$21.99';
+const twoProductPrice = '$43.98';
```

Comment:
I recommend, for more clarity, testing for `2 * singleProductPrice` instead of having this variable

Reply:
Done in 14c21452fcc11922efc41c33f6c1dedd086f2748. :slightly_smiling_face: